From 16eb431ab79ba3845f2701a289bd63a295db2e71 Mon Sep 17 00:00:00 2001
From: Girish Kumar M P <quic_gmp@quicinc.com>
Date: Tue, 7 May 2024 15:48:10 +0530
Subject: [PATCH] wifi: ath12k: Rename wbm2host-tx-completions-ring4 irq name.

Currently wbm2host-tx-completions-ring4 is mapped to incorrect interrupt
and tx-completion interrupts were not recieved as a result throughput
dropped to 0.

Fix by renaming the existing wbm2host-tx-completions-ring4 to
wbm2host-tx-completions-ath-ring4 and mapping it to correct interrupt.

patch-dependency: 710-001-ath12k-Add-AHB-driver-support-for-ipq5332.patch

Signed-off-by: Girish Kumar M P <quic_gmp@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/ahb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath12k/ahb.c b/drivers/net/wireless/ath/ath12k/ahb.c
index d156c04..3e7a8c6 100644
--- a/drivers/net/wireless/ath/ath12k/ahb.c
+++ b/drivers/net/wireless/ath/ath12k/ahb.c
@@ -82,7 +82,7 @@ static const char *irq_name[ATH12K_IRQ_NUM_MAX] = {
 	"host2tcl-input-ring3",
 	"host2tcl-input-ring2",
 	"host2tcl-input-ring1",
-	"wbm2host-tx-completions-ring4",
+	"wbm2host-tx-completions-ath-ring4",
 	"wbm2host-tx-completions-ring3",
 	"wbm2host-tx-completions-ring2",
 	"wbm2host-tx-completions-ring1",
-- 
2.34.1

